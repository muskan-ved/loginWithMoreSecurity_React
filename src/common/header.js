export const appAuthHeader = (auth_token) => {
    if (auth_token) {
      return { Authorization: `Bearer ${auth_token}` };
    } else {
      return {};
    }
  }

export function authHeader(tokens){

    if (tokens.headerAuthToken && tokens.headerLoginToken) {
        return { 'x-access-authorization': tokens.headerAuthToken, Authorization: `Bearer ${tokens.headerLoginToken}` };
      } else {
        return {};
      }
}

