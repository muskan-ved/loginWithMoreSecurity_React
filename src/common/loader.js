
import Loader from "react-js-loader";

export const LoaderFile = () => {
return (
        <div className={"row"}>
            <div className={"item"}>
                <Loader type="hourglass" bgColor={"#2065d1"} size={80} />
            </div>
        </div>
)
}