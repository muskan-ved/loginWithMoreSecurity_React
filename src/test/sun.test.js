import {ageCalculate} from '../common/ageCalculate'
import renderer from 'react-test-renderer';
import TestPage from 'src/Jest test/testPage';

test('my first test', () => {
    let expected = 10;
    expect(10).toBe(expected);
})

describe('second test', () => {

test('my age calculate test', () => {
        expect(ageCalculate("1998-10-03")).toEqual(24);
        expect(ageCalculate("1998-10-03")).not.toBe(23);
    })

})

describe('Snapshot test', () => {
it('renders correctly', () => {
 const tree = renderer
   .create(<TestPage page="www.softwaretestinghelp.com" hello='#'/>)
   .toJSON(); 
   console.log(tree,"-------")  
   expect(tree).toMatchSnapshot();
});
})


