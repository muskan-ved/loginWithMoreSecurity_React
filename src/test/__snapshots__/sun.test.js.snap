// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test renders correctly 1`] = `
<div>
  <div
    className="row"
  >
    <a
      className="container"
      href="www.softwaretestinghelp.com"
    >
      www.softwaretestinghelp.com
    </a>
    <div
      className="col-md-12"
    >
      <h1>
        Test Page
      </h1>
      <p>
        This is a test page 
        #
      </p>
    </div>
  </div>
</div>
`;
